unified-installer/astra-unified-installer.sh

```diff
@@ -1802,11 +1803,15 @@ step_check_k8s_version_in_range() {
     logheader $__DEBUG "Checking if Kubernetes version is within range ($minimum <=> $maximum)..."
 
     local current; current="$(kubectl version -o json 2> "$__ERR_FILE" | jq -r '.serverVersion.gitVersion' 2> /dev/null)"
-    local -r captured_err="$(get_captured_err)"
-    if [ -z "$current" ] || [ "$current" == null ] || [ -n "$captured_err" ]; then
+    local captured_err; captured_err="$(get_captured_err)"
+    if [ -z "$current" ] || [ "$current" == null ]; then
+        [ -z "$captured_err" ] && captured_err="unknown error"
```

Comment:
Previously, the check for `captured_err` was part of the first `if`, but warnings actually go through stderr (same as errors), so the following was being interpreted as an error:
```
WARNING: We cannot verify the version of the custom Astra Trident image you provided (docker.repo.eng.netapp.com/palice/trident:dev. Astra Control Provisioner support (23.10+) is not guaranteed.
```
This caused a successful check to instead manifest as a failure.